AsyncDisplayKit/ASDisplayNode.h

```diff
@@ -631,18 +632,13 @@ extern NSInteger const ASDefaultDrawingPriority;
 @interface ASDisplayNode (UIViewBridge)
 
 /**
- * Marks the view as needing display. Convenience for use whether the view / layer is loaded or not. Safe to call from a background thread.
+ * Marks the view as needing display. Convenience for use whether the view / layer is loaded or not. Safe to call
+ * from a background thread.
```

Comment:
I reverted it. Although I think we should have a line wrap with 120 points. @Adlai-Holler and me using this at the moment